vector/src/main/java/im/vector/app/features/notifications/NotifiableEventResolver.kt

```diff
@@ -94,7 +94,7 @@ class NotifiableEventResolver @Inject constructor(
     }
 
     suspend fun resolveInMemoryEvent(session: Session, event: Event, canBeReplaced: Boolean): NotifiableEvent? {
-        if (event.getClearType() != EventType.MESSAGE) return null
+        if (event.getClearType() !in listOf(EventType.MESSAGE, EventType.POLL_START)) return null
```

Comment:
I'm wondering if we could create an extension for this check and reuse in both places, could make adding more types easier in the future :crossed_fingers: 

```kotlin
fun Event.supportsNotification() = this.clearType in listOf(EventType.MESSAGE, EventType.POLL_START)
```

what do you think?

Reply:
Nice idea, done.